content/whats_on.md

```diff
@@ -1,13 +1,15 @@
 <div class="content" markdown="1">
+
 #### Weekend-Long Speedruning Stream
-![A speedrunner and his co-commentator at the event](http://localhost:8080/dist/img/speedrunning.jpg)
+![A speedrunner and their co-commentator at the event](http://localhost:8080/dist/img/speedrunning.jpg)
```

Comment:
MD: This title is incorrect for the image shown - it's a wide angle shot of the event. ![](https://wasd.warwick.gg/dist/img/speedrunning.jpg)

Reply:
Fixed in new commit